backend/server/src/main/java/ai/verta/modeldb/batchProcess/DatasetToRepositoryMigration.java

```diff
@@ -71,6 +71,7 @@ private DatasetToRepositoryMigration() {}
   public static void execute(int recordUpdateLimit) {
     DatasetToRepositoryMigration.recordUpdateLimit = recordUpdateLimit;
     config = App.getInstance().mdbConfig;
+    var executor = FutureExecutor.initializeExecutor(config.getGrpcServer().getThreadCount());
```

Comment:
Is this change related to the main purpose of the PR, or is it just cleanup? (Fine either way, I just want to check.)

Reply:
Reverted this change. it was not needed.